src/ctl/cart_ctl.c

```diff
@@ -377,18 +383,37 @@ parse_args(int argc, char **argv)
 	return rc;
 }
 
+void
+print_uri_cache(struct crt_ctl_get_uri_cache_out *out_uri_cache_args)
+{
+	struct crt_grp_cache    *grp_cache;
+	int			 count;
+	int			 i;
+
```

Comment:
This should check first out_uri_cache_args->rc and only print if rc == 0, else print error message and exit with error code